site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+  <div class="offcanvas-header p-0">
+    <button type="button" class="btn-close ms-auto" data-bs-target="#sideNavCollapsibleStatic" data-bs-dismiss="offcanvas" aria-expanded="true" aria-controls="sideNavCollapsibleStatic"><span class="visually-hidden">Close</span></button>
+  </div>
+  <div class="offcanvas-body h-100 p-0">
+    <nav class="side-nav side-nav-static" aria-label="Static collapsible side navigation inside an offcanvas">
+      <div class="side-nav-collapsible">
+        ...
+      </div>
+    </nav>
+  </div>
+</div>
+
+<div class="d-none d-lg-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+```
+
+## Scrollable
+
+Even if we handle the scroll in a particular way inside our predefined themes, you may want to change this behavior with the `.side-nav-scrollable`. If you do so, please make sure that every part is accessible through scroll (the icon part and the content one).
+
+<!-- Following the spec review
+
+Here are some examples with the [collapsible with drawer content](#collapsible-with-drawer-content), that default to a scroll on both parts (the icon part and the content part) excluding the title.
+
+Here is an example including the title on the right scroll box.
+
+<div class="bd-example p-lg-0 side-nav-example d-block d-lg-flex">
+  <button class="btn btn-primary d-lg-none" data-bs-target="#sideNavCwdc2" data-bs-toggle="offcanvas" aria-haspopup="true" aria-controls="sideNavCwdc2" id="sideNavCwdc2Toggle" onclick="boosted.Collapse.getOrCreateInstance('#sideNavCollapseCwdc2').show()">Test</button>
+  <div class="offcanvas-start offcanvas-lg d-flex" role="region" aria-labelledby="sideNavCwdc2Toggle" id="sideNavCwdc2">
+    <div class="offcanvas-header p-0 side-nav-content-with-drawer">
+      <button type="button" class="btn-close ms-auto" data-bs-dismiss="offcanvas" data-bs-target="#sideNavCwdc2" aria-expanded="true" aria-controls="sideNavCwdc2"><span class="visually-hidden">Close</span></button>
+    </div>
+    <div class="offcanvas-body p-0">
+      <nav class="side-nav" aria-label="Collapsible side navigation with drawer content with scroll boxes including the title">
+        <div class="side-nav-content-with-drawer">
+          <button class="side-nav-toggle collapsed" data-bs-toggle="collapse" data-bs-target="#sideNavCollapseCwdc2"><span class="visually-hidden">Open/Close side navigation</span></button>
+          <div class="side-nav-content">
+            <ul class="h-100 list-unstyled m-0 side-nav-dark side-nav-scrollable" role="tablist">
+              <li role="presentation" data-bs-toggle="tooltip" data-bs-title="Dashboard" data-bs-placement="right">
+                <button class="side-nav-item active" data-bs-toggle="tab" data-bs-target="#dashboardTab2" type="button" role="tab" aria-controls="dashboardTab2" aria-selected="true" onclick="boosted.Collapse.getOrCreateInstance('#sideNavCollapseCwdc2').show()">
```

Comment:
It seems that `aria-selected="<true | false >"` attribute is not allowed on a `button` element (https://www.w3.org/TR/html-aria/#docconformance).
You may need to double check with our expert and maybe make some changes.

Reply:
It's done automatically by Bootstrap Js, it might change if we have specific Js for this one.
Meanwhile, do you think it would be better to use `<a>`? I don't know what would be the great HTML here. I feel like the links are overkill since we don't change the page but I might be wrong.